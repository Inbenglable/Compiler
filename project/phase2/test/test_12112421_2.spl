int a(int b, struct A b){
    return 1;
}