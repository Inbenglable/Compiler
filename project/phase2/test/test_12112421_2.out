Error type 16 at line 1: undefined structure
Error type 3 at line 1: variable "b" is redefined in the same scope
